src/types/ListAuditLogResponse.ts

```diff
@@ -0,0 +1,5 @@
+export type ListAuditLogResponse<T> = {
```

Comment:
Let generalize this name to future proof it a bit -- how about `CursoredListResponse`, and stick in the `ListResponse` file?

Reply:
that's valid, although I feel like "ListAuditLog" is a bit more descriptive personally. Like when I see this in 4 months again it'll take me a little while to remember that "CursoredListResponse" is referring to the audit logs. so im gunna push through for now but am definitely open to changing when the need arises!